lib/dispatch-rider/logging/translator/base_translator.rb

```diff
@@ -0,0 +1,40 @@
+module DispatchRider
+  module Logging
+    class Translator
+
+      class BaseTranslator
+        def initialize(message, **_)
```

Comment:
I'm not sure if `**` works but I know the lone star works `*` for catching all unnamed variables. You could probably drop `_` here.